### ISSUE TITLE
GFXs don't work with new sprite system

### ISSUE THREAD

**Issue by fodinabor** [ivansafrin/Bitworld#1]
I wondered why I got a mainly white screen when building Bitworld with newest Polycode:
- the sprite system changed - the easiest would be to merge the Image files - but before I do this and create the sprites out of it I wanted to ask whether we are allowed (licensewise) to modify the images?

**Comment by fodinabor** [comment]
@ivansafrin as I wanted to do some work on this tomorrow I would love to have an answer when working with it ;-)

**Comment by fodinabor** [comment]
I just realized it didn't even use Sprites.. should I change it to use sprites or do you think this look comes from something else:
![image](https://cloud.githubusercontent.com/assets/5982050/5425312/b3407dc8-8311-11e4-80b4-48cb2b364108.png)

**Comment by chubchub5577** [comment]
As I am someone who is loving Polycode and loved this wonderful very pretty game, i would say thats a bug from the fact that the framework it self has changed so drastically. I really hope you fix this!

**Comment by fodinabor** [comment]
Yeah I'll try I think I got to know the bug a bit in the mean while - the same that hinders setImageCoordinate to be working..

**Comment by fodinabor** [comment]
Yep I found the issue - fixes incoming :-)
